ui/src/components/Voter/Voter.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './Voter.css';
+
+const Voter = (props) => {
+    return (
+        <tr className="poll-voter-row">
```

Comment:
I see this occur a lot around the codebase, and it's a valid way of picking up your props.
It might be easier, however, to destructure props from the getgo and then use the variables, something like this:
```js
const { ind, id, name, removeVoter } = props;
```